src/clj_jargon/users.clj

```diff
@@ -25,6 +25,12 @@
   (for [^UserGroup ug (.findUserGroupsForUser ug-ao user)]
     (.getUserGroupName ug)))
 
+(defn user-group-ids
+  "Returns a list of group IDs that the user is in."
+  [{^UserGroupAO ug-ao :userGroupAO} user]
+  (for [^UserGroup ug (.findUserGroupsForUser ug-ao user)]
+    (.getUserGroupId ug)))
```

Comment:
This was conveniently already provided; no need to do the `username->id` mapping if we just called it right initially!